src/Response/Search.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Envato\Response;
+
+class Search {
```

Comment:
There is going to be a bit of ambiguity here since people are able to search for items **and** comments via the API. I see a couple of ways we could approach this:

1) Have a single `search` method on the `ApiClient` that you tell which type you're after. Something like `search($type = 'item', $options - [])` would do. This would allow the first parameter to work out which `Response` class to send the data too  and the second to be the list of options you're after.
2) Be specific about the multiple endpoints. This would have two methods on the `ApiClient` (perhaps `item_search` and `comment_search`?) that would line up with the other methods and have a single `Response` class for it.

Thoughts?